.github/ISSUE_TEMPLATE/lint-request.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Lint request
+about: Request a new lint rule.
+title: ''
+labels: enhancement, lint request
+assignees: ''
+
+---
+
+**Describe the rule you'd like to see implemented**
+Include as much detail as you can.
```

Comment:
Perhaps add information about some of the most important details. For example "This should include a description of who would use the lint (such as if it's specific to a framework like Flutter), what conditions the rule should look for, and what conditions the rule should ignore."

Reply:
Great.  Done.  Thank you!